src/settings.rs

```diff
@@ -6,7 +6,7 @@ use storage::types::StorageConfig;
 #[derive(Debug, Deserialize, Clone)]
 pub struct ServiceConfig {
     pub host: String,
-    pub rest_port: u16,
+    pub http_port: u16,
     pub grpc_port: Option<u16>, // None means that gRPC is disabled
```

Comment:
I meant this comment to be in the `config/config.yaml`, users won't see comments in rust source

Reply:
Good point :man_facepalming: